styles.css

```diff
@@ -156,18 +159,30 @@ h2 {
 .img1 {
     aspect-ratio: auto 212 / 54;
 }
-#mail {
-    border-style: none;
-    border-radius: 25px;
+.inputwrapper {
+    width: 100%;
+    max-width: 520px;
     padding: 15px;
```

Comment:
1. use margin instead of padding
2. add height 48px;
3. remove all styles for flex (here better to hide button and duplicate html  code)